### ISSUE TITLE
[Bug]: Doorbell FPS setting ignored

### ISSUE THREAD

**Issue by butaca** [homebridge-eufy-security/plugin#125]
### What happened?

I'm running homebridge on a Raspberry PI 3 B+ so I'm tweaking the video settings because the default one doesn't run smoothly. These are my video settings right now:

"videoConfig": {
    "audio": true,
    "maxWidth": 640,
    "maxHeight": 480,
    "maxFPS": 12
}

The video stream starts the the requested resolution, but the FPS are fixed at 30:

7/23/2022, 7:19:52 PM] [EufySecurity-2.1.0-beta.12] INFO:  Timbre Starting video stream: 640x480, 30 fps, 299 kbps  (AAC-eld)

### Device Type

Doorbell (Supported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
7/23/2022, 7:19:52 PM] [EufySecurity-2.1.0-beta.12] INFO:  Timbre Starting video stream: 640x480, 30 fps, 299 kbps  (AAC-eld)
```

**Comment by thieren** [comment]
Hi @butaca 

I think I found the cause for this. Will try to fix it in the next beta release. Stay tuned!

**Comment by thieren** [comment]
This should be fixed with 2.1.0-beta.13

**Comment by butaca** [comment]
It works now! Thank you.
